packages/adaptive_scaffold/example/lib/adaptive_layout_demo.dart

```diff
@@ -6,21 +6,23 @@ import 'package:adaptive_scaffold/adaptive_scaffold.dart';
 import 'package:flutter/material.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(const _MyApp());
 }
 
-class MyApp extends StatelessWidget {
-  const MyApp({Key? key}) : super(key: key);
+class _MyApp extends StatelessWidget {
+  const _MyApp({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      home: const MyHomePage(),
+    return const MaterialApp(
+      home: MyHomePage(),
     );
   }
 }
 
+/// Public for testing purposes.
```

Comment:
I don't think you need this comment.

Reply:
I just had these since it was asking me to document the public classes. I could either remove the comments or turn them into something a bit more descriptive. Which do you think would be better?

Reply:
Something more descriptive.  Just describe what the class is for.  Besides, if you just remove them, it'll nag you about them until you at least add a comment telling it not to nag you.